command/meta.go

```diff
@@ -427,3 +438,197 @@ type funcVar func(s string) error
 func (f funcVar) Set(s string) error { return f(s) }
 func (f funcVar) String() string     { return "" }
 func (f funcVar) IsBoolFlag() bool   { return false }
+
+type UIRoute struct {
+	Path        string
+	Description string
+}
+
+type UIHintContext struct {
+	Command    string
+	PathParams map[string]string
+	OpenURL    bool
+}
+
+const (
+	// Colors and styles
+	resetter = "\033[0m"
+	magenta  = "\033[35m"
+	blue     = "\033[34m"
+	bold     = "\033[1m"
+
+	// Output formatting
+	uiHintDelimiter = "\n\n==> "
+	defaultHint     = "See more in the Web UI:"
+)
+
+var CommandUIRoutes = map[string]UIRoute{
+	"server members": {
+		Path:        "/servers",
+		Description: "View and manage Nomad servers",
+	},
+	"node status": {
+		Path:        "/clients",
+		Description: "View and manage Nomad clients",
+	},
+	"node status single": {
+		Path:        "/clients/:nodeID",
+		Description: "View client details and metrics",
+	},
+	"job status": {
+		Path:        "/jobs",
+		Description: "View and manage Nomad jobs",
+	},
+	"job status single": {
+		Path:        "/jobs/:jobID@:namespace",
+		Description: "View job details and metrics",
+	},
+	"job run": {
+		Path:        "/jobs/:jobID@:namespace",
+		Description: "View this job",
+	},
+	"alloc status": {
+		Path:        "/allocations/:allocID",
+		Description: "View allocation details",
+	},
+	"var list": {
+		Path:        "/variables",
+		Description: "View Nomad variables",
+	},
+	"var list prefix": {
+		Path:        "/variables/path/:prefix",
+		Description: "View Nomad variables at this path",
+	},
+	"var get": {
+		Path:        "/variables/var/:path@:namespace",
+		Description: "View variable details",
+	},
+	"var put": {
+		Path:        "/variables/var/:path@:namespace",
+		Description: "View variable details",
+	},
+	"job dispatch": {
+		Path:        "/jobs/:dispatchID@:namespace",
+		Description: "View this job",
+	},
+	"eval list": {
+		Path:        "/evaluations",
+		Description: "View evaluations",
+	},
+	"eval status": {
+		Path:        "/evaluations?currentEval=:evalID",
+		Description: "View evaluation details",
+	},
+	"deployment status": {
+		Path:        "/jobs/:jobID/deployments",
+		Description: "View all deployments for this job",
+	},
+}
+
+func (m *Meta) formatUIHint(url string, description string) string {
+	if description == "" {
+		description = defaultHint
+	}
+
+	description = fmt.Sprintf("%s in the Web UI:", description)
+
+	// Basic version without colors
+	hint := fmt.Sprintf("%s%s %s", uiHintDelimiter, description, url)
+
+	// If colors are disabled, return basic version
+	_, coloredUi := m.Ui.(*cli.ColoredUi)
+	if m.noColor || !coloredUi {
+		return hint
+	}
+
+	return fmt.Sprintf("%[1]s%[2]s%[3]s%[4]s%[5]s %[6]s%[7]s%[8]s",
+		bold,
+		magenta,
+		uiHintDelimiter[1:], // "==> "
+		description,
+		resetter,
+		blue,
+		url,
+		resetter,
```

Comment:
No need to change but at this point just doing `strings.Join([]string{ /* long list */ }, "")` is probably more clear.